modular_skyrat/master_files/code/modules/mob/mob_movement.dm

```diff
@@ -0,0 +1,21 @@
+/mob/living/carbon/verb/army_crawl()
+	set name = "Army Crawl"
+	set category = "IC"
+
+	var/mob/living/carbon/crawler = src
+
+	if(HAS_TRAIT(crawler, TRAIT_PRONE))
+		visible_message("[crawler] starts getting back up")
+		if(!do_after(crawler, 3 SECONDS))
+			return
+		SEND_SIGNAL(crawler, COMSIG_MOVABLE_REMOVE_PRONE_STATE)
+	else
+		visible_message("[crawler] begins to lower themself further")
+		if(!do_after(crawler, 3 SECONDS, extra_checks = CALLBACK(crawler, PROC_REF(can_army_crawl))))
+			if(!crawler.resting)
+				balloon_alert(crawler, "must be laying down")
```

Comment:
```suggestion
				balloon_alert(crawler, "must be laying down!")
```
Error balloon alerts end with an exclamation mark.

Reply:
Also, this procs if we're not resting AND if we've also failed the do_after - if the do_after succeeds, this doesn't proc. Move it over to right before the AddComponent call.

Reply:
the do_after can't succeed if you're standing up see - extra_checks = CALLBACK(crawler, PROC_REF(can_army_crawl)

Reply:
https://github.com/user-attachments/assets/fd65cfec-fc2d-486a-b15b-f98a4e0c8f07